content/en/_index.md

```diff
@@ -178,14 +203,22 @@ public static void exampleSnippet(String projectId, String filePath) {
     // Snippet content ...
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+// exampleSnippet demonstrates best practices.
+// The function prints to an io.Writer for testing purposes. Do not print to
```

Comment:
Just confirming that this comment is something to help the sample implementer and not something to be included in the samples themselves.

Reply:
Yes, that's correct.